tests/test_action_service.py

```diff
@@ -0,0 +1,28 @@
+from bot_python_sdk.action_service import ActionService
+from bot_python_sdk.action_service import ACTION_ID, DEVICE_ID, QUEUE_ID, VALUE, ALTERNATIVE_ID
+from unittest.mock import patch
+
+@patch('bot_python_sdk.configuration.Configuration.get_device_id')
+@patch('bot_python_sdk.key_generator.KeyGenerator.generate_uuid')
+def test_create_trigger_body(generate_uuid, get_device_id):
+
+	as_obj = ActionService()
+
+	uuid = 42
+	generate_uuid.return_value = uuid
+	device_id = 43
```

Comment:
The device_id should be a string value ('43')

Reply:
Thank you. I have now updated the device_id to a string value.